src/lambda/handler-runner/docker-runner/DockerContainer.js

```diff
@@ -344,19 +337,8 @@ export default class DockerContainer {
     return gateway.split("/")[0]
   }
 
-  async #ping() {
-    const url = `http://${this.#dockerOptions.host}:${this.#port}/2018-06-01/ping`
-    const res = await fetch(url)
-
-    if (!res.ok) {
-      throw new Error(`Failed to fetch from ${url} with ${res.statusText}`)
-    }
-
-    return res.text()
-  }
-
   async request(event) {
-    const url = `http://${this.#dockerOptions.host}:${this.#port}/2015-03-31/functions/${this.#functionKey}/invocations`
+    const url = `http://${this.#dockerOptions.host}:${this.#port}/2015-03-31/functions/function/invocations`
```

Comment:
is ${this.#functionKey} not working here?

Reply:
On the new images, to invoke a function is no needed to use the function key.

https://docs.aws.amazon.com/lambda/latest/dg/go-image.html#go-image-clients

Heres an example on the documentation. 

 ![Uploading Screenshot 2024-02-10 at 18.49.06.png…]()